### ISSUE TITLE
Error on https://uploads.ungrounded.net/ruffle_wrapper/ruffleembed.html

### ISSUE THREAD

**Issue by Robert9157** [ruffle-rs/ruffle#11301]
# Error Info
Error name: Error
Error message: panicked at 'No uniform for push constant', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-hal-0.16.0/src/gles/command.rs:725:17
Error stack:
```
Error: panicked at 'No uniform for push constant', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-hal-0.16.0/src/gles/command.rs:725:17
    at W.n.wbg.__wbg_new_87297f22973157c8 (https://uploads.ungrounded.net/ruffle_wrapper/lib/core.ruffle.53952f07cbd7713b7677.js:1:51142)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::he40396eea82bdc98 (wasm://wasm/02c5413a:wasm-function[8638]:0x835738)
    at std::panicking::rust_panic_with_hook::h7f7102b82d51338f (wasm://wasm/02c5413a:wasm-function[5634]:0x75925b)
    at core::panicking::panic_fmt::he4489d678d6570d5 (wasm://wasm/02c5413a:wasm-function[8487]:0x8309aa)
    at wgpu_hal::gles::command::<impl wgpu_hal::CommandEncoder<wgpu_hal::gles::Api> for wgpu_hal::gles::CommandEncoder>::set_push_constants::h26946696b41b0fd5 (wasm://wasm/02c5413a:wasm-function[3427]:0x6534f1)
    at <T as wgpu::context::DynContext>::command_encoder_end_render_pass::h83a504e227e766d7 (wasm://wasm/02c5413a:wasm-function[8021]:0x81b2f8)
    at <wgpu::RenderPass as core::ops::drop::Drop>::drop::hc7a554913ab621f9 (wasm://wasm/02c5413a:wasm-function[6779]:0x7af578)
    at ruffle_render_wgpu::surface::Surface::apply_color_matrix::h4a060f5012e5377c (wasm://wasm/02c5413a:wasm-function[939]:0x324e89)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::apply_filter::hb0702af0e53e8af3 (wasm://wasm/02c5413a:wasm-function[1344]:0x3f7164)
    at ruffle_core::avm2::globals::flash::display::bitmap_data::apply_filter::h16883bc5de8563b2 (wasm://wasm/02c5413a:wasm-function[1170]:0x3a1d4c)
```
AVM2 stack:
```
    at flash.display::BitmapData/flash::display::BitmapData::applyFilter()
    at Singleton::GameWorld/screenCap()
    at Systems::PlayerControlSystem/update()
    at Systems::System/_update()
    at Systems::System/_update()
    at Systems::System/_update()
    at Core/update()
    at Main/update()
```

# Player Info
Allows script access: true
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (NVIDIA Corporation, NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2, OpenGL 4.5.0)"
Adapter Device Type: Other
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS)
Available features: Features(TEXTURE_COMPRESSION_BC | CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 32768, max_texture_dimension_2d: 32768, max_texture_dimension_3d: 16384, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 640, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 550, height: 400, depth_or_array_layers: 1 }

# Page Info
Page URL: https://uploads.ungrounded.net/ruffle_wrapper/ruffleembed.html?v=1.0.67&browsermode=default&props=%7B%22ruffle%22%3A%22%2Fruffle_wrapper%2Flib%2Fruffle.js%3F1685448187%22%2C%22public_path%22%3A%22%2Fruffle_wrapper%2Flib%2F%22%2C%22swf%22%3A%22https%3A%2F%2Fuploads.ungrounded.net%2F634000%2F634358_DeadZed2.swf%3F1567074177%22%2C%22vars%22%3A%7B%22NewgroundsAPI_PublisherID%22%3A1%2C%22NewgroundsAPI_SandboxID%22%3A%22647661c5e2ac1%22%2C%22NewgroundsAPI_SessionID%22%3A%22%22%2C%22NewgroundsAPI_UserName%22%3A%22%26lt%3Bdeleted%26gt%3B%22%2C%22NewgroundsAPI_UserID%22%3A0%2C%22ng_username%22%3A%22%26lt%3Bdeleted%26gt%3B%22%7D%2C%22width%22%3A640%2C%22height%22%3A480%2C%22icon%22%3A%22https%3A%2F%2Fpicon.ngfiles.com%2F634000%2Fflash_634358_card.png%3Ff1601414790%22%2C%22warnOnUnsupportedContent%22%3Afalse%7D
SWF URL: https://uploads.ungrounded.net/634000/634358_DeadZed2.swf?1567074177

# Browser Info
User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36
Platform: Linux x86_64
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-05-30
Channel: nightly
Built: 2023-05-30T00:21:44.167Z
Commit: 8d6c08df4d10fb2f77511af3d605c61a5b94b7bc
Is extension: false

# Metadata
width: 640
height: 480
frameRate: 30
numFrames: 2
swfVersion: 13
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 12746204

**Comment by Robert9157** [comment]
Seems reproducible with Dead Zed 2 and Storm Ops 3 both from the author.

**Comment by n0samu** [comment]
Should be fixed by #11753 and #11750
